src/Microsoft.Diagnostics.Runtime/src/Utilities/PEImage/ImageResourceDirectoryEntry.cs

```diff
@@ -21,44 +22,42 @@ namespace Microsoft.Diagnostics.Runtime.Utilities
     /// set to indicate this.  Otherwise the high bit is clear and the offset
     /// field points to a resource data entry.
     /// </summary>
-    internal struct IMAGE_RESOURCE_DIRECTORY_ENTRY
+    [StructLayout(LayoutKind.Sequential)]
```

Comment:
Has this struct change been tested?  Is the layout before/after the `StructLayout` and field rearrange the same?

Reply:
Ah these are the only fields, the rest are properties.

Reply:
Struct layouts are sequential by default.

See Remarks in https://docs.microsoft.com/dotnet/api/system.runtime.interopservices.structlayoutattribute